### ISSUE TITLE
Detect crashed scripts and exit run

### ISSUE THREAD

**Issue by runspired** [TracerBench/tracerbench#186]
If an error prevents the page load from completing, the run will hang forever. We should add a configurable timeout for preventing this.

**Comment by lynchbomb** [comment]
This should be resolved with the latest 4.0 major release of tracerbench.

**Comment by lynchbomb** [comment]
FWIW 
```
--sampleTimeout=sampleTimeout
      [default: 30] The number of seconds to wait for a sample.
```
